picard/coverart/image.py

```diff
@@ -480,3 +480,6 @@ def __init__(self, filepath, types=None, comment='',
         super().__init__(url=url, types=types, comment=comment)
         self.support_types = support_types
         self.support_multi_types = support_multi_types
+        path = self.url.toLocalFile()
+        with open(path, 'rb') as file:
+            self.set_data(file.read())
```

Comment:
I think loading the data here makes sense, but this needs fixing a couple of tests that currently fail due to this. The `LocalFileCoverArtImageTest` currently works without files, should be updated to work on temporary files.